misc/packaging/README.md

```diff
@@ -0,0 +1,117 @@
+# ubuntuSoS
+this scripts show how to build acrn-hypervisor and acrn-kernel deb package
+
+1. setup native ubuntu
+
+install python3
```

Comment:
Is this the only missing package requirement?  It's best to give the commands to do the install, ``sudo apt install python3`` (and will that get all the expected python utils)?

Reply:
add sudo apt install python3